accounts/checkers_test.go

```diff
@@ -523,7 +523,9 @@ func testSendPayment(t *testing.T, uri string) {
 	errFunc := func(err error) {
 		lndMock.mainErrChan <- err
 	}
-	service, err := NewService(t.TempDir(), errFunc)
+	store, err := NewBoltStore(t.TempDir(), DBFilename)
```

Comment:
Should we close the `store` at the end of the tests here? I see this is changed in the next commits, with addition of the interface, so maybe it'd be worth adding a `cleanup` return arg for that interface that closes the store?

Reply:
in the commit that introduces the `NewTestDB` calls, it's taken care of in the clean up there. So I think your concerns are addressed by the end of the PR no? (let me know if i misunderstand).